src/View/View.php

```diff
@@ -84,7 +85,7 @@ public function template($template = null)
 
     public function render(): string
     {
-        GlobalRuntimeState::resetGlobalState();
+        ViewRendering::dispatch($this);
```

Comment:
Is there a clear reason or use case where this event would be helpful?

If not, I'd say remove it until someone actually needs it.